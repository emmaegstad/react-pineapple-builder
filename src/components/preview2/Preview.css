.Preview {
  margin-bottom: 2vh;
}

.charSection img {
  height: 100px;
  width: auto;
}
