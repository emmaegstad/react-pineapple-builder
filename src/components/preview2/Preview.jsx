import React from 'react';
import './Preview.css';

export default function Preview({ top, middle, bottom }) {
  return (
    <div className="Preview">
      <div className="charSection top">
        <img className="image" src={`${process.env.PUBLIC_URL}/assets/${top}.png`} alt="top" />
      </div>
      <div className="charSection middle">
        <img
          className="image"
          src={`${process.env.PUBLIC_URL}/assets/${middle}.png`}
          alt="middle"
        />
      </div>
      <div className="charSection bottom">
        <img
          className="image"
          src={`${process.env.PUBLIC_URL}/assets/${bottom}.png`}
          alt="bottom"
        />
      </div>
    </div>
  );
}
