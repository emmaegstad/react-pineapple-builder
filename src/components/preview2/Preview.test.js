import { render, screen } from '@testing-library/react';
import Preview from './Preview';

test('renders the Editor component', () => {
  const container = render(<Preview bottom="bottom-1" middle="middle-1" top="top-1" />);
  expect(container).toMatchSnapshot();
});
