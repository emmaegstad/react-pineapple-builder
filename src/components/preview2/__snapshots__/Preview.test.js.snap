// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Editor component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="Preview"
      >
        <div
          class="charSection top"
        >
          <img
            alt="top"
            class="image"
            src="/assets/top-1.png"
          />
        </div>
        <div
          class="charSection middle"
        >
          <img
            alt="middle"
            class="image"
            src="/assets/middle-1.png"
          />
        </div>
        <div
          class="charSection bottom"
        >
          <img
            alt="bottom"
            class="image"
            src="/assets/bottom-1.png"
          />
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="Preview"
    >
      <div
        class="charSection top"
      >
        <img
          alt="top"
          class="image"
          src="/assets/top-1.png"
        />
      </div>
      <div
        class="charSection middle"
      >
        <img
          alt="middle"
          class="image"
          src="/assets/middle-1.png"
        />
      </div>
      <div
        class="charSection bottom"
      >
        <img
          alt="bottom"
          class="image"
          src="/assets/bottom-1.png"
        />
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
